### DISCUSSION TITLE
Dota crash after patch 7.32 E

### DISCUSSION THREAD

**Discussion by Larchompus** [ValveSoftware/Dota2-Gameplay#7771] [Q&A]
Hi , players. I have a rly big problem and don't know what to do. Dota 2 crashes during the game withot any logs or mistakes. And i cannot point "why". Thats why I am always in LP, cause after recon.  dota will give me some minutes to play and again Dota 2 crashes.
Thats why I need help cause I rly don't understand where the conflict zone in the gaiming process.

Thanks

My spec:
Intel-i5 13600K 
3080 TI Zotac gaming
16 GB (x2) Kingston FURY- DDR4
Z690M AORUS ELITE AX 

Some solutions that I've tried:
1. Clean the cash 
2. Reload full game 
3. Upgrade video drivers and Direct X

**Comment by kortana45** [comment]
According to my knowledge in the Steam library, right-click on Dota 2, go to Properties, select the "Local Files" tab, and click on "Verify Integrity of Game Files." This process checks for any corrupted or missing game files and replaces them if necessary. 


[www.prepaidgiftbalance.com](https://www.prepaidgiftbalance.vip/)
